docs/docusaurus/docs/oss/guides/connecting_to_your_data/fluent/database/gcp_deployment_patterns_file_bigquery.py

```diff
@@ -259,7 +261,10 @@
 
 
 # <snippet name="docs/docusaurus/docs/oss/guides/connecting_to_your_data/fluent/database/gcp_deployment_patterns_file_bigquery.py add_or_update_expectation_suite">
-context.add_or_update_expectation_suite(expectation_suite_name="test_bigquery_suite")
+try:
+    context.suites.add(ExpectationSuite(name="test_bigquery_suite"))
+except DataContextError:
+    ...
```

Comment:
Are we okay with this for now or should we have an idomatic pattern for idempotent creation/retrieval?

Reply:
I think we're okay for now. We've said we don't want to continue with the `add_or_update_` pattern any more. I do find the try/accept stuff pretty ugly though.